src/main/java/io/fabric8/elasticsearch/plugin/KibanaUserReindexAction.java

```diff
@@ -168,38 +167,7 @@ public void apply(String action, ActionResponse response, @SuppressWarnings("raw
     private GetResult buildNewResult(GetResponse response) {
         String index = getIndex(response);
         String replacedIndex = response.getIndex();
-
-        // Check for .kibana.* in the source
-        BytesReference replacedContent = null;
-        if (!response.isSourceEmpty()) {
-            String source = response.getSourceAsBytesRef().toUtf8();
-            String replaced = source.replaceAll(replacedIndex, index);
-            replacedContent = new BytesArray(replaced);
-        }
-
-        // Check for .kibana.* in the fields
-        Map<String, GetField> responseFields = response.getFields();
```

Comment:
@jcantrill, in the new code, the private variable `responseFields` is initialized to an empty `hashmap` at line 45 of `GetResultBuilder.java`, which avoid the `NullPointer` error at line 79 in `GetResultBuilder.java`, is that the correct understanding?